xapian-core/cluster/euclidian_sim.cc

```diff
@@ -0,0 +1,75 @@
+#include <config.h>
```

Comment:
I used this file to separate the implementations from cluster.cc. I've included the "xapian/cluster.h" which declares the EuclidianDistance class. So should I create a new header file for this with the classes used for document similarity calculation?